openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -474,6 +475,7 @@ def process(self, instance):
                 "FTRACK_SERVER": os.environ.get("FTRACK_SERVER"),
             }
 
+        deadline_publish_job_id = None
         if submission_type == "deadline":
```

Comment:
Out of curiosity - when is this **not** the case whenever the Deadline plugins run?

Reply:
It seems like this is only the case where `muster` would be used for farm publishing.